src/kubernetes/mysql/mysql-secrets.yaml

```diff
@@ -6,3 +6,4 @@ metadata:
     app: mysql
 data:
   mysql-root-password: eW91cnBhc3N3b3Jk
+  mysql-root-username: cm9vdA==
```

Comment:
I realized we need to add this in the template config as well. will fix it.

Reply:
Great Catch!!!!!   Thanks!